lib/options/rule-delimiter.js

```diff
@@ -0,0 +1,47 @@
+'use strict';
+
+var gonzales = require('gonzales-pe');
+
+module.exports = {
+    name: 'rule-delimiter',
+
+    syntax: ['scss', 'css', 'less'],
+
+    runBefore: 'strip-spaces',
+
+    setValue: function(value) {
+        var regex = /^[\n]*$/;
+
+        if (typeof value === 'number') {
+            value = new Array(Math.round(value) + 2).join('\n');
+        } else if (typeof value === 'string' && !value.match(regex)) {
+            throw new Error('The option only accepts the newline character or numbers (\\n), you provided %s', value);
```

Comment:
I guess this is related to the first comment. I am not sure how I feel about this being supported, it feels like the job of a minifier tbh. Thoughts?